include/wlr/types/wlr_foreign_toplevel_management_v1.h

```diff
@@ -126,4 +127,14 @@ void wlr_foreign_toplevel_handle_v1_set_activated(
 void wlr_foreign_toplevel_handle_v1_set_fullscreen(
 	struct wlr_foreign_toplevel_handle_v1* toplevel, bool fullscreen);
 
+/* Send a parent change event to all clients that hold a handle to
```

Comment:
I wouldn't say `send a parent change` because we are setting the toplevel parent. The change is also sent to any clients which come afterwards (so it should be `set` not `send`)